### DISCUSSION TITLE
Clean/plain text on paste

### DISCUSSION THREAD

**Discussion by sahudhirendra** [sibiraj-s/ngx-editor#411] [Ideas]
When doing copy paste from another document, its takting over the format of the other document. To have all text clean,is there any attribute or method available with this editor?

Thanks,
Dhirendra

**Comment by sibiraj-s** [comment]
You can create a prosemirror plugin that transforms pasted html into however you want. You might want to look for events

- [transformPastedText](https://prosemirror.net/docs/ref/#view.EditorProps.transformPastedText)
- [transformPastedHTML](https://prosemirror.net/docs/ref/#view.EditorProps.transformPastedHTML)
- [transformPasted](https://prosemirror.net/docs/ref/#view.EditorProps.transformPasted)
